cli.js

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env node
 
-const useYarn = require('.')
+var fs = require('fs')
+var path = require('path')
+var argv = require('minimist')(process.argv.slice(2));
+var useYarn = require('.')
 
-useYarn()
+var message
+if (argv.f) {
+  message = fs.readFileSync(argv.f, 'utf8')
+}
+
+if (!message && argv.m) {
+  message = argv.m
+}
+
+if (!message) {
+  message = fs.readFileSync(path.resolve(__dirname, 'message.txt'), 'utf8')
+}
+
+useYarn(message)
```

Comment:
Do you think we should accept file paths and support the default message with the programmatic API?

Reply:
We probably need to change the function signature to allow this, but I have no use case for it. The consumer could also read the file themselves and then just pass on the message.